template/docker-entrypoint.sh

```diff
@@ -379,6 +391,7 @@ _main() {
 			mysql_note "MySQL init process done. Ready for start up."
 			echo
 		fi
+		mysql_socket_fix
```

Comment:
For `mysqlsh` to work in `docker-entrypoint-initdb.d` scripts too, this needs to happen after `docker_init_database_dir` too, right? (maybe this becomes `else` instead of outside the `if`?)